src/libostree/ostree-repo.h

```diff
@@ -767,14 +767,17 @@ typedef struct {
   gboolean enable_fsync;  /* Deprecated */
   gboolean process_whiteouts;
   gboolean no_copy_fallback;
-  gboolean unused_bools[7];
+  gboolean force_copy;
```

Comment:
I may break this out as a separate PR too with exposure via the cmdline as well.